Add FAQ section

duranmla opened the pull request:
**What:**
Add a new section of FAQ

**Why:**
Closes https://github.com/debtcollective/student-debt-campaign/issues/8

**How:**
Add a new section of FAQ using [`<details>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details) tag.

**Notice:** as previous created section I haven't add the CMS connection with the content. On top of that **this PR needs https://github.com/debtcollective/student-debt-campaign-netlify/pull/4** in order the FAQ render as show in the screenshots below.

Ultimately, I have choose to not add any extra polyfill for the details tag since the IE browsers difference will be that the user is not able to toggle the information but it is still usable, while people with more widely user browsers will have a enhance experience

**Notice:** This PR also add a fix within the notification component to bring it up.

## Screenshots
![http://recordit.co/evKJJeiNiR](http://recordit.co/evKJJeiNiR.gif)

[iPad](https://user-images.githubusercontent.com/1425162/63636568-53244200-c671-11e9-94f5-8bbffe3b2b34.png)
[IE Edge](https://user-images.githubusercontent.com/1425162/63636583-8070f000-c671-11e9-825a-f44159c47720.png)
[FireFox](https://user-images.githubusercontent.com/1425162/63636587-88c92b00-c671-11e9-8003-2495d4069b5a.png)
[Android Chrome](https://user-images.githubusercontent.com/1425162/63636589-8e267580-c671-11e9-8b11-694f5610c752.png)

duranmla committed:
feat(FAQ): add FAQ section with some of the values it has

duranmla committed:
feat(FAQ): add support of <details> customization on FF

duranmla committed:
feat(FAQ): clean <details> customization on IE browsers

duranmla committed:
feat(FAQ): remove horizontal pad on items for mobile devices

orlando commented:
Let's try https://github.com/rstacruz/details-polyfill. If it works OTB then we leave it, if not we remove it and leave Internet Explorer rendering like it is now.

duranmla committed:
feat(faq): add polyfill for details

duranmla committed:
fix(notification): bring component up in z layer

this will avoid the notification to overlap with the content while user scrolls

duranmla commented:
After the polyfill introduced at 6a46348

It looks like this:
![image](https://user-images.githubusercontent.com/1425162/63831371-d1812c80-c96e-11e9-93ea-5c2605e6cb05.png)

duranmla committed:
build: require dependencies only if window exists
